freud/diffraction.py

```diff
@@ -65,6 +55,8 @@ def min_valid_k(self):
 
     def _repr_png_(self):
         try:
+            import freud.plot
+
```

Comment:
To appease the pre-commit and the linter you want to revert the changes on the conditional imports to the code that is current with the main branch.